ui/src/external/monaco.markdown.theme.ts

```diff
@@ -0,0 +1,25 @@
+import {MonacoType} from 'src/types'
+
+const THEME_NAME = 'markdownTheme'
+
+function addTheme(monaco: MonacoType) {
+  monaco.editor.defineTheme(THEME_NAME, {
+    base: 'vs-dark',
+    inherit: false,
+    rules: [],
```

Comment:
...and add them to the rules here. take a look at the monaco.toml.theme.ts and monaco.flux.theme.ts in the same folder